google/cloud/bigtable/internal/mutate_rows_limiter.cc

```diff
@@ -86,8 +87,12 @@ std::shared_ptr<MutateRowsLimiter> MakeMutateRowsLimiter(
   };
   sleeper = internal::MakeTracedSleeper(
       options, std::move(sleeper), "gl-cpp.bigtable.bulk_apply_throttling");
-  // TODO(#12959) - use a real sleeper.
-  auto async_sleeper = [](duration) { return make_ready_future(); };
+  auto async_sleeper = [cq = std::move(cq), options](duration d) mutable {
+    // Capture `options` by value because the source `options` gets moved from.
```

Comment:
Nit:  This almost goes without saying when the lifetime of the lambda exceeds the lifetime of the call.

Reply:
PS:  It's also an indication that `MakeMutateRowsLimiter()` perhaps should take its `Options` by value.

Reply:
Good point. Moved the copy from the lambda to the factory function, as you suggested.